src/pages/Chat/hooks/useChatData.ts

```diff
@@ -0,0 +1,52 @@
+import { useTranslation } from "react-i18next";
+import {
+  useGetMessagesNotificationsQuery,
+  useGetRoomsQuery,
+} from "services/notifications/setNotificationsAPI";
+import { IUser, useGetOwnUserQuery } from "services/user/setUserAPI";
+import { IChatUser } from "services/notifications/chatTypes";
+
+const useChatData = () => {
+  const { t } = useTranslation();
+
+  const { data: roomsList } = useGetRoomsQuery();
+
+  const { data: user } = useGetOwnUserQuery();
+
+  const { data: notificationsResponse } = useGetMessagesNotificationsQuery();
+
+  const getRoomNotifications = (roomId: number): number[] | undefined => {
+    if (notificationsResponse && notificationsResponse.notifications) {
+      return notificationsResponse.notifications
+        .filter((notification) => notification.roomId === roomId)
+        .map((notification) => notification.id);
+    }
+    return undefined;
```

Comment:
can we just `return` without any value?

Reply:
I believe it is more explicit. as long as the badge takes either number or undefined.